### ISSUE TITLE
[v1alpha2] New UI

### ISSUE THREAD

**Issue by richardsliu** [kubeflow/katib#443]

**Comment by andreyvelich** [comment]
As we discussed today, after we will merge this PR: https://github.com/kubeflow/katib/pull/438, we should start to work on v1alpha2 version of that New UI.
Is that correct @richardsliu @johnugeorge ?

**Comment by johnugeorge** [comment]
i was suggesting to directly make changes for v1alpha2 once grpc apis are merged(this week or early next week). UI has dependency only with APIs and they can even be tested with mock data.   

I feel that it might be a waste of effort by fixing for v1alpha1 and then again making new changes to v1alpha2 again.

**Comment by andreyvelich** [comment]
@johnugeorge Alright

**Comment by jlewi** [comment]
Any update on this issue?

**Comment by andreyvelich** [comment]
@jlewi We have finished UI for v1alpha1 version, we just need to use new API for v1alpha2 and it will work. 
@richardsliu Should we do it with 0.6 release or we can finish it to the middle of May?

**Comment by richardsliu** [comment]
@andreyvelich The PR can be merged as soon as it is ready. It will be released as part of 0.6.

**Comment by jlewi** [comment]
@andreyvelich What is the remaining work to close out this issue? Are we waiting on #486 to be merged?

Do we need to create a separate issue to update the kustomize manifests to use the new UI?

**Comment by andreyvelich** [comment]
@jlewi Add comment here: https://github.com/kubeflow/katib/pull/486#issuecomment-491658792.
I will add kustomize manifest after I finish the UI.

**Comment by gaocegege** [comment]
Did #486 solve the issue?

**Comment by andreyvelich** [comment]
Yes. We can close this issue. I will test our new UI.
/close
